test/SolidityNameAndTypeResolution.cpp

```diff
@@ -321,6 +323,23 @@ BOOST_AUTO_TEST_CASE(comparison_bitop_precedence)
 	BOOST_CHECK_NO_THROW(parseTextAndResolveNames(text));
 }
 
```

Comment:
There is no real good fit anywhere. The ABIJson as the name implies only tests the generated JSON. Maybe the compiler tests would also make sense? But the reason I used name and type resolution is that this has to happen only after NameAndTypeResolution and as such thought it a good enough fit.